python/paddle/static/nn/control_flow.py

```diff
@@ -1138,28 +1144,30 @@ def cond(pred, true_fn=None, false_fn=None, name=None, return_names=None):
             #     return 3, 2
             #
 
-            def true_func():
-                return paddle.full(shape=[1, 2], dtype='int32',
-                                   fill_value=1), paddle.full(shape=[2, 3],
-                                                              dtype='bool',
-                                                              fill_value=True)
+            >>> def true_func():
+            ...     return paddle.full(shape=[1, 2], dtype='int32',
+            ...                         fill_value=1), paddle.full(shape=[2, 3],
+            ...                                                     dtype='bool',
+            ...                                                     fill_value=True)
```

Comment:
这部分代码写得好乱，手动格式化一下吧

```suggestion
            >>> def true_func():
            ...     return paddle.full(
            ...         shape=[1, 2],
            ...         dtype='int32',
            ...         fill_value=1
            ...     ), paddle.full(
            ...         shape=[2, 3],
            ...         dtype='bool',
            ...         fill_value=True
            ...     )
```

下同